lib/features/sks-menu/presentation/widgets/sks_menu_loading.dart

```diff
@@ -0,0 +1,41 @@
+import "package:flutter/material.dart";
+import "package:lottie/lottie.dart";
+
+import "../../../../../theme/app_theme.dart";
+import "../../../../../utils/context_extensions.dart";
+import "../../../../config/ui_config.dart";
+import "../../../../gen/assets.gen.dart";
+
+class SKSMenuLottieAnimation extends StatelessWidget {
+  const SKSMenuLottieAnimation({
+    super.key,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Column(
+        mainAxisAlignment: MainAxisAlignment.center,
+        children: [
+          SizedBox.square(
+              dimension: 200,
+              child: Lottie.asset(
+                Assets.animations.sksClosed,
+                fit: BoxFit.cover,
+                repeat: false,
+                frameRate: const FrameRate(LottieAnimationConfig.frameRate),
+                renderCache: RenderCache.drawingCommands,
+              ),
+            ),
+          Align(
+              child: Text(
```

Comment:
Maybe it'd be good to add  detailed error message (error message that comes from api). Maybe in small print and in different color.